index.js

```diff
@@ -174,7 +172,8 @@ module.exports = function(S) {
                   return response || responses['default'];
                 }
 
-                fun.run( _this.evt.stage, _this.evt.region, event ).then( (res)=> {
+                let runtime = fun.getRuntime();
+                runtime.run(fun,  _this.evt.stage, _this.evt.region, event).then( (res)=> {
```

Comment:
Does it change anything?
https://github.com/serverless/serverless/blob/master/lib/Function.js#L237-L239